versioned_docs/version-5.x/header-buttons.md

```diff
@@ -79,7 +79,7 @@ function HomeScreen({ navigation }) {
 
 You can change the label behavior with `headerBackTitle` and `headerTruncatedBackTitle` ([read more](stack-navigator.html#headerbacktitle)).
 
-To customize the back button image, you can use [headerBackImage](stack-navigator.html#headerbackimage).
+To customize the back button image, you can use [headerBackImage](https://reactnavigation.org/docs/stack-navigator/#headerbackimage).
```

Comment:
```suggestion
To customize the back button image, you can use [headerBackImage](/docs/stack-navigator/#headerbackimage).
```
We should use relative link instead of absolute link